tests/foreman/api/test_convert2rhel.py

```diff
@@ -288,6 +290,7 @@ def test_convert2rhel_oracle_with_pre_conversion_template_check(
     result = module_target_sat.api.JobInvocation(id=job['id']).read()
     assert result.succeeded == 1
     # execute job 'Convert 2 RHEL' on host
+    assert oracle.execute('yum -y update kernel-uek*').status == 0
```

Comment:
Does this fix the current OEL issues with C2R, could you run PRT for both CentOS and OEL?

Reply:
There is some issue with running RHEL7 which is unrelated and I would take that in different PR. This PR covers automation of given feature which is working.

Reply:
yeah, but this line looks unrelated to this PR, so either could you remove it from here and add it in the different PR along with the OEL C2R fixes?

Reply:
Yes. This is unrelated. I will remove it and take this to another PR.